README.md

```diff
@@ -84,6 +84,7 @@ mapper#map(data, type, mapperOptions)
     - _(required)_ `offset` _(integer)_
     - _(required)_ `limit` _(integer)_
     - _(optional)_ `total` or `rowCount` _(integer)_
+  - _(optional)_ `attrWhitelist` _(string[])_: in accordance with the JSON API spec, the mapper automatically filters out any foreign keys from the primary payload. We currently match on `id`, `-id`, `_id`, `modelId` (camelCase), `-type`, `_type` and `Type` (camelCase). If you need to have an attribute that matches one of the aforementioned patterns pass through, you can add it to the whitelist.
```

Comment:
the documentation says `attrWhitelist` and the code says `attrsWhitelist` (note the **`s`**)